examples/notebooks/run_from_xlsx.ipynb

```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PoPS Border simulation using XLSX file upload\n",
+    "\n",
+    "## Test example \n",
+    "\n",
+    "First, we complete the Excel template `data/small_config.xlsx` with the minimal configuration parameter required to run the simulation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load required functions and packages\n",
+    "\n",
+    "from popsborder.simulation import run_simulation\n",
+    "from popsborder.inputs import load_configuration\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Excel configuration should include all parameters related to:\n",
+    "1. [consignments](../../docs/consignments.md) (what is imorted, from where, in what amounts), \n",
+    "2. [contaminants](../../docs/contamination.md) (proportion and distribution of contaminated consignments, boxes, and items), and \n",
+    "3. [inspection](../../docs/inspections.md) (sampling strategies and rates)\n",
+    "\n",
+    "There is a help file (linked above) for each of these sections that goes into greater detail about how to define the parameters."
```

Comment:
Maybe also include text before every code cell describing the step. The two following code cells can be one introduced in a separate paragraph here as _Now we load the configuration from a file:_ or something like that.